docker-compose.yml

```diff
@@ -1,17 +1,23 @@
 version: '3'
 services:
   web:
-    image: docker-registry.tools.wmflabs.org/toollabs-static-web
+    image: docker-registry.tools.wmflabs.org/toollabs-php-web
     ports:
       - 8888:80
     command: lighttpd -D -f /etc/lighttpd/lighttpd.conf
     volumes:
       - ./etc/lighttpd/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
-      - ./:/var/www:cached
+      - ./client:/var/www/client:cached
+      - ./server:/var/www/server:cached
   build:
     image: docker-registry.tools.wmflabs.org/toollabs-nodejs-web
-    working_dir: /code
+    working_dir: /code/client/
     command:
       - ./bin/watch
     volumes:
-      - ./:/code:cached
+      - ./:/code
+  composer:
```

Comment:
Is composer on toolforge? or is it in the `docker-registry.tools.wmflabs.org/toollabs-php-web` image?

Reply:
now that you mentioned it, toollabs-php-web image has composer. We could move this to be a command on web instead. Although, I kinda like it separate